### PR TITLE
Themes

### PR DESCRIPTION
**Pull request by dudethatbe**
Doesn't do anything yet, but here's the dropdown. Not sure how to select all of the DOM elements and change their class quite yet ...

### CODE REVIEW

**Comment by mcfiredrill** [comment]
closes https://github.com/datafruits/datafruits/issues/289

File: app/components/datafruits-player.hbs [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<audio preload="" id="radio-player" class="hidden">
```
Comment (mcfiredrill) [inline app/components/datafruits-player.hbs:1]: Did you mean to move this template from `app/templates/components` to `app/components` ??

File: app/components/theme-selector.hbs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<label
+  id="theme-selector-label"
+  for="theme-selector"
+>
+  theme
```
Comment (mcfiredrill) [inline app/components/theme-selector.hbs:5]: We should use a translated string here.

File: app/components/theme-selector.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import classic from 'ember-classic-decorator';
+import Component from '@ember/component';
+
+@classic
```
Comment (mcfiredrill) [inline app/components/theme-selector.js:4]: This should convert to a glimmer component easily....fine to do it later on though.

File: app/tailwind/config.js [hunk 4]
```diff
@@ -25,8 +25,16 @@ module.exports = {
         'df-green': '#5fdc82',
         'df-green-dark': '#2bc456',
       },
+      zIndex: {
+        '-10': '-10',
```
Comment (mcfiredrill) [inline app/tailwind/config.js:29]: Is this being used? I couldn't find it...

**Comment by mcfiredrill** [comment]
*this is amazing*

themes

File: app/tailwind/config.js (hunk 4 above)
Comment (dudethatbe) [reply app/tailwind/config.js:29]: Oh woops that was when i was trying to use the divs and had to send them to the background. I can remove that easily

File: app/components/datafruits-player.hbs (hunk 1 above)
Comment (dudethatbe) [reply app/components/datafruits-player.hbs:1]: Yeah, but I think that was unrelated, and probably doesn't need to happen in this PR :sweat_smile:

File: tests/integration/components/theme-selector-test.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | theme-selector', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
```
Comment (mcfiredrill) [inline tests/integration/components/theme-selector-test.js:9]: `this.setThemetest = () => {};`

File: app/templates/home/about.hbs [hunk 6]
```diff
@@ -208,7 +208,7 @@
   <br>
   <br>
   <br>
-  <a href="http://datavegetables.club" class="classic:text-df-blue blm:text-black trans:text-df-pink hover:text-df-yellow text-shadow-none">
+  <a href="http://datavegetables.club" class="classic:text-df-blue blm:text-black trans:text-df-blue hover:text-df-yellow text-shadow-none">
```
Comment (mcfiredrill) [inline app/templates/home/about.hbs:211]: :sparkles:

**Comment by dudethatbe** [comment]
thanks for all the help @mcfiredrill 😁

**Comment by mcfiredrill** [comment]
Stoked to launch this! Thanks for all your work on it. @dudethatbe

**Pull request merged**